protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -660,10 +660,17 @@ service TradingDataService {
     //
     // Estimate the margin that would be required for submitting this order
     rpc EstimateMargin(EstimateMarginRequest) returns (EstimateMarginResponse) {
-      option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+        option deprecated = true;
+        option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
 	tags: "Orders";
       };
-    }
+    };
+
+    // Estimate position
+    //
+    // Estimate the margin that would be required for maintaining the specified position.
+    // If the optional collateral available argument is supplied, the response also contains the estimate of the liquidation price.
+    rpc EstimatePosition(EstimatePositionRequest) returns (EstimatePositionResponse);
```

Comment:
Can we add a tag to this API so that it gets put in the correct category in the auto-generated docs:
```
option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
	tags: "???";
      };
```
`EstimateMargin` was under `Orders` but we also have a category for `Positions` which might be better?